lib/infoblox.rb

```diff
@@ -13,7 +13,7 @@
 end
 
 module Infoblox
-  WAPI_VERSION   = '1.0'
+  WAPI_VERSION   = '1.1'
```

Comment:
Yes, I think that would be an acceptable approach (using the environment variable).  I don't have access to the 1.1 API to test this - though  I'll certainly accept a PR.